hacks/generate-controller-registration.sh

```diff
@@ -38,15 +38,15 @@ providerConfig:
     ### "image" overwrites extension (for seed) and "images" overwrite installation/agent (for shoot) images defined in charts/images.yaml
     #image:
     #  repository: v2.isvimgreg.com/gardener-extension-cri-resmgr
-    #  tag: mybranch
+    #  tag: latest
```

Comment:
Is it compatible with what is written in the documentation? Maybe it would be more helpful to leave it as it was with "mybranch"